superset-frontend/src/features/databases/DatabaseModal/ExtraOptions.tsx

```diff
@@ -571,6 +571,23 @@ const ExtraOptions = ({
             )}
           </div>
         </StyledInputContainer>
+        <StyledInputContainer css={no_margin_bottom}>
+          <div className="input-container">
+            <IndeterminateCheckbox
+              id="disable_drill_to_detail"
+              indeterminate={false}
+              checked={!!extraJson?.disable_drill_to_detail}
+              onChange={onExtraInputChange}
+              labelText={t('Disable drill to detail')}
+            />
+            <InfoTooltip
+              tooltip={t(
+                `Disables the drill to detail feature for this database. This might be useful if
+                the database does not support atomic-level queries or contains large amounts of data.`,
```

Comment:
```suggestion
                `Disables the drill to detail feature for this database.`,
```

Reply:
@michael-s-molina I was trying to think about what to say here, i.e., making mention to the fidelity of the data, but couldn't find the right words. I think it's likely really a case by case basis and thus we probably don't need to provide suggestions as to why this might be needed.